homeassistant/helpers/device_registry.py

```diff
@@ -156,7 +156,7 @@ class _EventDeviceRegistryUpdatedData_Remove(TypedDict):
 
     action: Literal["remove"]
     device_id: str
-    device: DeviceEntry
+    device: dict[str, Any]
```

Comment:
Har vi ingen TypedDict för DeviceEntry?

Reply:
No, it's just a `dict[str, Any]`: https://github.com/home-assistant/core/blob/cc227b2317cdf99b17c77cbe75c1eabca992111a/homeassistant/helpers/device_registry.py#L359

Reply:
Ok.